core/iwasm/fast-jit/jit_regalloc.c

```diff
@@ -350,6 +354,29 @@ is_alloc_candidate(JitCompContext *cc, JitReg reg)
             && (!jit_cc_is_hreg(cc, reg) || !jit_cc_is_hreg_fixed(cc, reg)));
 }
 
+#ifdef VREG_DEF_SANITIZER
+static void
+check_vreg_definition(RegallocContext *rc, JitInsn *insn)
+{
+    unsigned j;
+    JitRegVec regvec = jit_insn_opnd_regs(insn);
+    unsigned first_use = jit_insn_opnd_first_use(insn);
+    JitReg *regp;
+
+    /* to check if a vr has a definition before it is refernced */
```

Comment:
refernced -> referenced
How about change to "check if a vr has a definition before it is referenced"? No need to add `to` as it is what we are currently doing.